.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "salesforce.salesforcedx-vscode",
+        "redhat.vscode-xml",
+        "orta.vscode-jest",
+        "firsttris.vscode-jest-runner"
+    ]
```

Comment:
I recommend adding EditorConfig (`"EditorConfig.editorconfig"`). We already include the `.editorconfig` resource.

Reply:
Good point. We should remove `.editorconfig` as, moving forward, Prettier will be the default recommended formatter.

Reply:
Will this result in a workflow whereby a dev’s local changes are formatted very differently until they run prettier? Things like matching tab indent go a long way

Reply:
When they use the Prettier extension for VS Code there won't be a difference.

Reply:
It's my understanding that without editorconfig, even with the Prettier extension (which I don't see here in the recommended list) you still need to edit your vscode settings to set `editor.formatOnSave` to true to get the auto-formatting during development. 

Otherwise we fallback to `lint-staged` running prettier at git commit time. Please correct me if you see different behavior but that's what I've experienced locally.

Reply:
Having an EditorConfig file in the project root doesn't do anything, unless someone has the `EditorConfig` extension installed (that's different to i. e. `Visual Studio` which has built-in support). As, moving forward, Prettier will be the recommended formatter, it's not doing anything.

And yes, the VS Code user needs to enable how s/he wants to have formatting when editing/saving manually.